src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php

```diff
@@ -44,6 +44,11 @@
 		<?php
 		}
 		?>
+				<div class="tjdashboardloadingwidget">
```

Comment:
Better use `<script id="hello" type="text/template"></script>` instead of this ?